zcash_client_backend/Cargo.toml

```diff
@@ -17,49 +17,72 @@ rust-version = "1.56.1"
 development = ["zcash_proofs"]
 
 [dependencies]
+zcash_address = { version = "0.2", path = "../components/zcash_address" }
```

Comment:
A problem that I encountered in the zcash-light-client-ffi crate is that these path-based dependencies can cause problems if you neglect to override one of them with a patch version, even if you're not directly depending upon the crate. Is it possible to remove the path dependency, and instead have that scoped to the patch directives in the workspace file?